plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -1823,7 +1823,7 @@ public static function activated_plugin( $filename ) {
 			}
 
 			// No more sites available in this subscription.
-			if ( $_sub['sites_max'] && $_sub['sites_active'] >= $_sub['sites_max'] ) {
+			if ( $_sub['maxed'] ) {
```

Comment:
We're generally trying to be a little more defensive in our coding. It seems like this data is coming from a remote source, and is also filterable (because it's obtained using either the Transient API or Remote HTTP API). In addition to checking it is truthy, should we also check if it is actually set?

```suggestion
			if ( isset( $_sub['maxed'] ) && $_sub['maxed' ) {
```